docs/040_pepr-tutorials/030_create-pepr-operator.md

```diff
@@ -628,18 +609,23 @@ function configmap(instance: WebApp) {
 }
 ```
 
+Our job is to make the deployment of the WebApp simple. Instead of having to keep track of the versions and revisions to all of the Kubernetes manifest required for the WebApp, and rolling pods and updating configMaps, the deployer now only needs to focus on the `WebApp` instance. The controller will reconcile instances of the operand (WebApp) against the actual cluster state to reach the desired state.
```

Comment:
```suggestion
Our job is to make the deployment of the WebApp simple. Instead of having to keep track of the versions and revisions of all of the Kubernetes Objects required for the WebApp, rolling pods and updating configMaps, the deployer now only needs to focus on the `WebApp` instance. The controller will reconcile instances of the operand (WebApp) against the actual cluster state to reach the desired state.
```